tools/run_tests/run_interop_tests.py

```diff
@@ -800,9 +851,15 @@ def cloud_to_prod_jobspec(language,
         transport_security_options = [
             '--custom_credentials_type=google_default_credentials'
         ]
+    elif transport_security == 'compute_engine_channel_creds' and str(
```

Comment:
the previous elif branch is `transport_security == 'google_default_credentials'` but you're removing that option.

Reply:
This was a bit weird. I've removed `google_default_credentials` as an option for a value of `--transport_security` argument to the whole script, but it's still a possible value for the parameter to `cloud_to_prod_jobspec`, because we're still appending "cloud_to_prod_jobspec with google default credentials" [to the list of test cases](https://github.com/grpc/grpc/blob/master/tools/run_tests/run_interop_tests.py#L1408).